tasks/validate_host_names.yaml

```diff
@@ -1,24 +1,10 @@
 - fail:
```

Comment:
i like to name my tasks so there is some context in the output eg. `name: Validate values for DNS compatibility`

Reply:
ACK, good catch